aws-observability-terraform/source-module/outputs.tf

```diff
@@ -58,6 +58,26 @@ output "elb_auto_enable_stack" {
   description = "AWS CloudFormation stack for ALB Auto Enable access logs."
 }
 
+output "clasic_lb_sns_topic" {
```

Comment:
correct spelling for all occurrences - clasic to classic

Reply:
Done